### ISSUE TITLE
header>nav>span>a

### ISSUE THREAD

**Issue by farishkash** [nielsjlarsen/Tea-Cozy#4]
This is useful and proper if you have a page with multiple <a> elements all over the web page.  Since you only use this in one location, it is always better to go the simpliest route and directly target this selector.

```
a {
	display: inline-block;
	text-decoration: underline;
	text-align: right;
	color: seashell;
	padding: 10px;
}
```

**Comment by nielsjlarsen** [comment]
I agree but on the other hand I'm trying to get used to a re-usable coding style that will scale well.
So you're saying keep it simple is the best way to go - always?
